src/main/java/com/celements/javascript/ExtJsFileParameter.java

```diff
@@ -0,0 +1,73 @@
+package com.celements.javascript;
+
+import static com.google.common.base.Preconditions.*;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import com.celements.web.plugin.cmd.AttachmentURLCommand;
+
+public final class ExtJsFileParameter {
+
+  @NotNull
+  private String jsFile;

```

Comment:
It doesn't make sense to annotate an uninitialised field, which is by default null, as `@NotNull`. Instead of annotating the fields add annotations consistently to the getters/setters.